client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -65,8 +87,28 @@ class ActivityLogDay extends Component {
 			ts_end_site_day: tsEndOfSiteDay,
 			utc_date: moment.utc( tsEndOfSiteDay ).format( 'YYYY-MM-DD' ),
 		} );
+
+		this.setState( {
+			dayExpanded: true,
+		} );
 	};
 
+	closeDayOnly = () =>
+		this.setState( {
+			rewindHere: false,
+			dayExpanded: false,
+		} );
+
+	closeDayAndRewindDialog = () => {
+		const { recordTracksEvent, rewindRequestDismiss, siteId } = this.props;
+		recordTracksEvent( 'calypso_activitylog_restore_cancel' );
+		rewindRequestDismiss( siteId );
```

Comment:
The last thing I think I would recommend on this PR is that we don't have to make the component aware of the metrics we are tracking. The fact that we're tracking the cancellation is incidental to the job the UI is performing and we have a nice way of [extracting that logic](https://github.com/Automattic/wp-calypso/tree/110106d322fd8138e9f9d203ac46a0518df2fd82/client/state/analytics), which again can make for easier testing as we don't have to test both that the button does its job _and_ the metrics are tracking.

```js
const mapDispatchToProps = dispatch => ( {
	cancelRewind: siteId => dispatch( withAnalytics(
		recordTracksEvent( 'calypso_activitylog_restore_cancel' ),
		rewindRequestDismiss( siteId )
	) )
} )
```

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/18722/commits/6f343353a11e557c8c5cfa349ae6890ef4418424
Thanks for the help with this.